### PR TITLE
Small fix for image buttons in navigation bar

### PR DESCRIPTION
**Pull request by sarahscott**

### CODE REVIEW

File: Example/Artsy-UIButtons.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -389,7 +389,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "diff \"${PODS_ROOT}/../Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n";
+			shellScript = "diff \"${PODS_ROOT}/../Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [[ $? != 0 ]] ; then\n    cat << EOM\nerror: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\nEOM\n    exit 1\nfi\n";
```
Comment (sarahscott) [inline Example/Artsy-UIButtons.xcodeproj/project.pbxproj]: Is this what we want or should I check it out?

File: Example/Podfile.lock [hunk 2]
```diff
@@ -46,4 +46,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: db5f7c1679c58f02e076b485442c8d0f1a385dd8
 
-COCOAPODS: 1.1.0.rc.2
```
Comment (sarahscott) [inline Example/Podfile.lock]: Ditto

File: Example/Artsy-UIButtons.xcodeproj/project.pbxproj (hunk 1 above)
Comment (alloy) [inline Example/Artsy-UIButtons.xcodeproj/project.pbxproj]: Yeah you can revert this. In fact, unless you need to make any dependency changes, you don’t need to run `pod install` at all anymore, provided you do checkout the submodule. If you haven’t done so yet:

```
$ cd Example
$ rm -rf Pods
$ git reset HEAD Pods
$ cd ..
$ git submodule update --init
```

This is from the top of my head. If it doesn’t work, maybe just make a new clone of the repo and run the last command and apply your code change there.

**Comment by sarahscott** [comment]
All set!

**Comment by alloy** [comment]
:+1:

**Pull request merged**